### PR TITLE
VaNotify - Update callback integration doc

### PR DESCRIPTION
**Pull request by ianperera**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- *This work is behind a feature toggle (flipper): NO
- *(Summarize the changes that have been made to the platform) Documentation Update
- *(If bug, how to reproduce)*
- *(What is the solution, why is this the solution?)*
- *(Which team do you work for, does your team own the maintenance of this component?)VA Notify - yes, my team owns this module
- *(If introducing a flipper, what is the success criteria being targeted?)*

## Related issue(s)

- vets-api callback support documentation
[#1410](https://github.com/department-of-veterans-affairs/vanotify-team/issues/1410)

## Testing done

- [ ] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
- *If this work is behind a flipper:*
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (modules/va_notify/README.md)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Comment by rmtolmach** [comment]
your branch was 44 commits behind so I updated it just to be safe.

**Review by rmtolmach — DISMISSED** [review]
Awesome, this will be really helpful 👍

File: modules/va_notify/README.md [hunk 1]
```diff
@@ -111,14 +111,131 @@ Using option #2:
...
+---
 
-A successful request to VA Notify API does not necessarily mean that your intended recipient will actually receive the notification due to the errors mentioned above. Please set up delivery status callbacks, so you can track the success or failure of your notification in real time. VA Notify provides aggregated metrics, but does not provide historical Veteran level metrics.
+#### Contact Us
 
-To track the delivery status of individual notifications you will need to set up service callbacks. These callbacks will allow you to determine the delivery status of each notification that you send. Here is some [documentation](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/va-notify#delivery-status-callbacks) regarding the callbacks that VA Notify provides (not `vets-api` specific).
+If you need any further clarification or help during the integration process, feel free to reach out:
 
-Please note, currently callbacks are limited to 1 per service. If you are in a shared service or need different callback webhooks by use case please contact VA Notify. We are actively working on an enhancement to support this!
+- Slack Channel: #va-notify-public
```
Comment (nathanbwright) [inline modules/va_notify/README.md:241]: Can you make this a link?

Comment (ianperera) [reply modules/va_notify/README.md:241]: Fixed. I tried different variations of linkage.

**Review by nathanbwright — APPROVED** [review]

**Review by stevenjcumming — APPROVED** [review]

**Pull request merged**